changes/api/+8b59c9eb.added.md

```diff
@@ -0,0 +1 @@
+Expanded logic to auto clean unused database entries to job templates and job submissions
```

Comment:
Now that we have a PR number, please, rename the file to 769.added.md

Reply:
Done!